crates/bevy_a11y/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "bevy_a11y"
+version = "0.9.0"
+edition = "2021"
+description = "Provides accessibility support for Bevy Engine"
+homepage = "https://bevyengine.org"
+repository = "https://github.com/bevyengine/bevy"
+license = "MIT OR Apache-2.0"
+keywords = ["bevy", "accessibility", "a11y"]
+
+[dependencies]
+# bevy
+bevy_app = { path = "../bevy_app", version = "0.9.0" }
+bevy_derive = { path = "../bevy_derive", version = "0.9.0" }
+bevy_ecs = { path = "../bevy_ecs", version = "0.9.0" }
+
+accesskit = "0.10"
```

Comment:
This does add a non-trivial number of new dependencies. On linux we now have 358 dependencies, up from 291. That is 61 new deps: a 23% increase.

Obviously accessibility is important. But numbers that big deserve increased scrutiny, even for important features. Where are these deps coming from? What purpose do they serve? Do we need them? What are the compile time implications? What do the dep counts look like on other platforms?

Reply:
Also binary-size implications, especially on WASM. I'm guessing right now WASM is largely unaffected, as these deps appear to be coming from the platform adapters and it doesn't look like there is a WASM adapter.

Reply:
In addition to the practical / technical issues mentioned above, this is also an optics issue. Bevy is already considered to be "big" and that is a turn off for some people. Substantial dep count increases won't help our case here.

It is definitely hard to add new features without adding new deps. Bevy needs to add new features, therefore our deps _will_ go up. But for all of the reasons listed in this thread we should be working extra hard to drive these numbers down.